package.json

```diff
@@ -35,15 +35,22 @@
     "start:fr": "ng serve --aot --i18nFile=src/locale/messages.fr.xlf --i18nFormat=xlf --locale=fr",
     "build:fr": "ng build --prod --i18nFile=src/locale/messages.fr.xlf --i18nFormat=xlf --locale=fr",
     "extract": "ng xi18n --outputPath=src/locale",
-    "build:ssr": "npm run build:client-and-server-bundles && npm run webpack:server",
+    "build:ssr": "npm run fix-window-print && npm run fix-colour-scss && npm run build:client-and-server-bundles && npm run webpack:server && npm run fix-server-js-syntax",
+    "build:ssr-mac": "npm run fix-window-print-mac && npm run fix-colour-scss-mac && npm run build:client-and-server-bundles && npm run webpack:server && npm run fix-server-js-syntax-mac",
     "serve:ssr": "node dist/server.js",
     "build:client-and-server-bundles": "ng build --prod --no-progress && ng run ccd-case-management-web:server:production --no-progress",
     "webpack:server": "webpack --config webpack.server.config.js --colors",
     "sonar-scan": "NODE_PATH=. sonar-scanner -X",
     "stub-api": "json-server -p 3453 --delay 800 --routes stubs/api/aggregated.routes.json --watch stubs/api/aggregated.api.json",
     "stub-functional": "json-server -p 3453 --delay 800 --routes stubs/functional/aggregated.routes.json --watch stubs/functional/aggregated.api.json",
     "snyk-protect": "snyk protect",
-    "prepare": "yarn run snyk-protect"
+    "prepare": "yarn run snyk-protect",
+    "fix-server-js-syntax": "sed -i \"s|@hmcts/media-viewer;|'@hmcts/media-viewer';|g\" ./dist/server.js && uname -a",
+    "fix-server-js-syntax-mac": "uname -a && sed -i \"\" \"s|@hmcts/media-viewer;|'@hmcts/media-viewer';|g\" ./dist/server.js",
+    "fix-colour-scss": "sed -i \"s|@error \\\"Unknown colour \\`#{\\$colour}\\`\\\";|@return \\$colour;|g\" ./node_modules/@hmcts/media-viewer/assets/govuk-frontend/helpers/_colour.scss",
+    "fix-colour-scss-mac": "sed -i \"\" \"s|@error \\\"Unknown colour \\`#{\\$colour}\\`\\\";|@return \\$colour;|g\" ./node_modules/@hmcts/media-viewer/assets/govuk-frontend/helpers/_colour.scss",
+    "fix-window-print": "find ./node_modules/@hmcts/media-viewer/ -type f -name '*.js' -exec sed -i \"s|printWindow.print();|;|g\" {} +;",
+    "fix-window-print-mac": "find ./node_modules/@hmcts/media-viewer/ -type f -name '*.js' -exec sed -i \"\" \"s|printWindow.print();|;|g\" {} +;"
   },
```

Comment:
`ng serve` or `yarn start` is failing after running `yarn install`.

Further, I suggest other developers as well @mario-paniccia @andrewfolga to review these lines of scripts particularly in `package.json` to avoid any cascading impacts in production.